entry_types/paged/app/views/pageflow_paged/pages/_page.html.erb

```diff
@@ -10,3 +10,11 @@
   <% end %>
   <a class="to_top" href="#content"><%= t('pageflow.public.goto_top') %></a>
 <% end %>
+
+<% if entry.feature_state('page_level_caching') %>
+  <%= cache [entry.locale, page] do %>
+    <%= content_for :page_content %>
+  <% end %>
+<% else %>
+  <%= content_for :page_content %>
+<% end %>
```

Comment:
I'd suggest to move this code into a helper method that wraps the `cache` call. This would help avoid the rather unidiomatic usage of `content_for`. Feature still needs translations (i.e. `pageflow.page_level_caching.feature_name`), but otherwise looks fine.

Reply:
We could also consider deploying the change from a temporary edge branch and only merge it to master once we know whether we actually need it. This would make these concerns less relevant. In this case, I'd still like to check the implementation of `content_for` to make sure it does not impose its own performance costs.

Reply:
Thanks for the reminder to use a helper! Yes, I think I'll push the change to an edge branch.

Do we need a spec for this, or is that also in case we actually need it? I tested the change in development, and am waiting for CI jobs to finish before preparing the edge deploy

Reply:
As long as it's not supposed to be merged, manual testing is fine with me.